zerver/tests/test_middleware.py

```diff
@@ -139,3 +139,12 @@ def test_nonexistent_page(self) -> None:
             [],
             # Test that our open graph logic doesn't throw a 500
             404)
+
+    def test_login_page_simple_description(self) -> None:
+        realm = get_realm('zulip')
+
+        self.check_title_and_description(
+            '/login/',
+            realm.name,
+            [' '.join(realm.description.split())],
```

Comment:
It's probably worth spelling out the realm description in the test just for clarity.